### PR TITLE
changes to make app work without google-services.json

### PR DESCRIPTION
**Pull request by ravjain-adb**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -50,7 +50,7 @@ captures/
 .externalNativeBuild
 
 # Google Services (e.g. APIs or Firebase)
-google-services.json
+#google-services.json
```
Comment (emdobrin) [inline .gitignore:53]: let's keep ignoring this

File: Sample-App/app/src/main/java/com/adobe/marketing/mobile/sampleapp/MainApp.java [hunk 2]
```diff
@@ -76,19 +76,23 @@ public void call(Object o) {
...
                         }
-
-                        // Get new FCM registration token
-                        String token = task.getResult();
-                        MobileCore.setPushIdentifier(token);
-                    }
-                });
+                    });
+        } catch (IllegalArgumentException e) {
+            Log.e(LOG_TAG, "Error while fetching token. Check if google-services.json is added");
```
Comment (emdobrin) [inline Sample-App/app/src/main/java/com/adobe/marketing/mobile/sampleapp/MainApp.java:95]: let's print the error message too, e.g.:
`Log.e(LOG_TAG, "IllegalArgumentException - Check if google-services.json is added and is correctly configured. \nError message: " + e.getLocalizedMessage());`

**Comment by ravjain-adb** [comment]
@emdobrin done with the updates

File: Sample-App/app/build.gradle [hunk 3]
```diff
@@ -37,7 +37,7 @@ dependencies {
     implementation 'com.adobe.marketing.mobile:edgeconsent:1.0.0'
     implementation 'com.adobe.marketing.mobile:edgeidentity:1.0.0'
     implementation 'com.adobe.marketing.mobile:analyticsedge:1.+'
-    implementation 'com.adobe.marketing.mobile:messaging:1.0.0-beta-1'
+    implementation 'com.adobe.marketing.mobile:messaging:1.0.0-beta-2-SNAPSHOT'
```
Comment (emdobrin) [inline Sample-App/app/build.gradle:40]: Let's revert this, we should only use stable versions in this app.

**Review by emdobrin — APPROVED** [review]
Looks good, with a comment

**Pull request merged**